source/src/ports/generic_networkhandler.c

```diff
@@ -29,8 +29,8 @@
 #define MAX_NO_OF_TCP_SOCKETS 10
 
 /** @brief Ethernet/IP standard port */
-extern const int kOpenerEipIoUdpPort = 0x08AE;
-const int kOpenerEthernetPort = 0xAF12;
+extern const unsigned short kOpenerEipIoUdpPort = 0x08AE;
```

Comment:
unsigned short has no guarantee on its length, therefore I would prefer uint16_t, which should also be available in recent versions of VS and would also satisfy the POSIX interface